test/topology_custom/test_multidc.py

```diff
@@ -0,0 +1,207 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test functionality on the cluster with different values of the --smp parameter on the nodes.
+"""
+import sys
+sys.path.insert(0, sys.path[0] + "/test/cql-pytest")
+import nodetool
+import logging
+import time
```

Comment:
These are libraries from the stdlib, they should be in a first block separated from the 3rd-party libraries by one empty line.